spec/controllers/work_packages/bulk_controller_spec.rb

```diff
@@ -643,5 +643,23 @@ def send_destroy_request
         expect(response).to redirect_to(project_work_packages_path(work_package1.project))
       end
     end
+
+    context "with children work packages following each other" do
+      before_all do
+        work_package1.update(subject: "parent", schedule_manually: false)
+        work_package2.update(subject: "predecessor child", parent: work_package1, schedule_manually: true)
+        work_package3.update(subject: "successor child", parent: work_package1, schedule_manually: false)
+        create(:follows_relation, predecessor: work_package2, successor: work_package3)
+      end
+
+      let(:params) { { "ids" => [work_package1.id, work_package2.id, work_package3.id] } }
+
+      it "deletes them all without errors" do
+        send_destroy_request
```

Comment:
Might be a bit redundant, but since the error states that we are explicitly testing for not raising an error, WDYT about adding this to explicitly check that we do not raise an error:

```suggestion
        expect { send_destroy_request }.not_to raise_error
```

Of course, the result would be the same as the test would also fail with the raising, but it might make it a bit more obvious what we are explicitly expecting from the test

Reply:
Good point. That indeed reads better.